<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Osciloscope and Function generator GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="RefreshAvailableBoards_pushButton">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="AvailableBoards_comboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="connect_pushButton">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Vmax</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="Vmax_doubleSpinBox">
        <property name="maximum">
         <double>3.200000000000000</double>
        </property>
        <property name="value">
         <double>3.200000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Time Sample In ms</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="TimeSample_spinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCustomPlot" name="graphPlot_QCustomPlot" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="Channel1_label">
               <property name="text">
                <string>Channel 1</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="Wave_Ch1_label">
               <property name="text">
                <string>Wave</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="Wave_Ch1_comboBox">
               <item>
                <property name="text">
                 <string>Disable Channel</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Sine Wave</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Triangle Wave</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Saw Tooth</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Square Wave</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="Amplitude_Ch1_label">
               <property name="text">
                <string>Amplitude</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="Amplitude_Ch1_doubleSpinBox">
               <property name="maximum">
                <double>3.200000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.250000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="Frequency_Ch1_label">
               <property name="text">
                <string>Frequency</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="Frequency_Ch1_spinBox">
               <property name="maximum">
                <number>5000</number>
               </property>
               <property name="singleStep">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="PhaseShift_Ch1_label">
               <property name="text">
                <string>Phase Shift</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="PhaseShift_Ch1_spinBox">
               <property name="maximum">
                <number>359</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="DutyCycle_Ch1_label">
               <property name="text">
                <string>Duty Cycle</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="DutyCycle_Ch1_spinBox">
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="CurrentConfiguration_Ch1_label">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Applied Configurations</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="WaveCurrentSetting_Ch1_label">
             <property name="text">
              <string>Wave</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="AmplitudeCurrentSetting_Ch1_label">
             <property name="text">
              <string>Amplitude</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="FrequencyCurrentSetting_Ch1_label">
             <property name="text">
              <string>Frequency</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="PhaseShiftCurrentSetting_Ch1_label">
             <property name="text">
              <string>Phase Shift</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="DutyCycleCurrentSetting_Ch1_label">
             <property name="text">
              <string>Duty Cycle</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="Channel2_label">
             <property name="text">
              <string>Channel 2</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="Wave_Ch2_label">
               <property name="text">
                <string>Wave</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="Wave_Ch2_comboBox">
               <item>
                <property name="text">
                 <string>Disable Channel</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Sine Wave</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Triangle Wave</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Saw Tooth</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Square Wave</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="Amplitude_Ch2_label">
               <property name="text">
                <string>Amplitude</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="Amplitude_Ch2_doubleSpinBox">
               <property name="maximum">
                <double>3.200000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.250000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="Frequency_Ch2_label">
               <property name="text">
                <string>Frequency</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="Frequency_Ch2_spinBox">
               <property name="maximum">
                <number>5000</number>
               </property>
               <property name="singleStep">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="PhaseShift_Ch2_label">
               <property name="text">
                <string>Phase Shift</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="PhaseShift_Ch2_spinBox">
               <property name="maximum">
                <number>359</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="DutyCycle_Ch2_label">
               <property name="text">
                <string>Duty Cycle</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="DutyCycle_Ch2_spinBox">
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="CurrentConfiguration_Ch2_label">
             <property name="text">
              <string>Applied Configurations</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="WaveCurrentSetting_Ch2_label">
             <property name="text">
              <string>Wave</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="AmplitudeCurrentSetting_Ch2_label">
             <property name="text">
              <string>Amplitude</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="FrequencyCurrentSetting_Ch2_label">
             <property name="text">
              <string>Frequency</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="PhaseShiftCurrentSetting_Ch2_label">
             <property name="text">
              <string>Phase Shift</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="DutyCycleCurrentSetting_Ch2_label">
             <property name="text">
              <string>Duty Cycle</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="ApplyFunctionGeneratorConfigurations_pushButton">
      <property name="text">
       <string>Apply Function Generator Configurations</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1164</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
